src/index.js

```diff
@@ -20,5 +21,6 @@ export {
   Menu,
   MenuGroup,
   MenuItem,
+  Pagination,
```

Comment:
I would expect this entire thing to be called "Pager", but that's maybe just me

Reply:
Yep, good point.